tests/unittests/test_null_cell_format.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+from tap_google_sheets import schema
+
+SHEET = {
+    "properties":{
+        "sheetId":1822945984,
+        "title":"Sheet26",
+        "index":1,
+        "sheetType":"GRID",
+        "gridProperties":{
+            "rowCount":20,
+            "columnCount":1
+        }
+    },
+    "data":[
+        {
+            "rowData":[
+                {
+                    "values":[
+                        {
+                            "formattedValue":"Column1",
+                        }
+                    ]
+                },
+                {
+                    "values":[
+                        {  
+                            "effectiveFormat": {
+                                "numberFormat": {
+                                    "type": None,
+                                }
+                            }
+                        }
+                    ]
+                }
+            ]
+        }
+    ]
+}
+
+class TestNullCellFormat(unittest.TestCase):
+    
+    def test_null_datetime_effectiveFormat(self):
+        """
+        Test when no value is given in second row of date-time, discovery is locking at 'effectiveFormat'.
+        And returns type and format in schema according to that.
+        """
+        sheet = SHEET
```

Comment:
This seems like a redundant assignment since `SHEET` is an dict object so any changes to `sheet` after assignment will update `SHEET` as well. 

Since `SHEET` is a global object it may accidentally affect other test execution as well.

Reply:
We are just changing one key in all the tests, and the function assignment also does not change the sheet, just gets the data from the sheet